client/__tests__/index.spec.js

```diff
@@ -0,0 +1,7 @@
+
+
+describe('My Test Suite', () => {
+  it('My Test Case', () => {
+    expect(true).toEqual(true);
```

Comment:
omg, sooo true!

Reply:
LMAO, LGTM though! Legit true equals to true. Can confirm 👍